run_release.py

```diff
@@ -179,15 +175,6 @@
 """
 
 
-@dataclass
-class Task:
```

Comment:
`Task` isn't used anywhere in `release.py` that I can see, is there a reason we moved it there?

Reply:
Yeah, it's used in typing of ReleaseShelf and it refers to ReleaseShelf itself. So it needs to be in the same spot.